yb-voyager/src/srcdb/postgres.go

```diff
@@ -30,7 +35,9 @@ func (pg *PostgreSQL) Connect() error {
 }
 
 func (pg *PostgreSQL) CheckRequiredToolsAreInstalled() {
-	checkTools("pg_dump", "strings", "pg_restore")
+	checkTools("strings")
+	// pgDumpPath, _ := GetAbsPathOfPGCommand("pg_dump")
```

Comment:
Get rid of the commented lines?

Reply:
Sorry this was an old commit. Just pushed the latest changes.

Reply:
@kneeraj I did test it by adding paths to both pg_dump binary versions 11 and 14 in the PATH. It chooses version 14 and proceeds. I basically replaced the pg_dump in /usr/bin to version 11 and added the absolute path to version 14 in the path variable.